tests/helpers/mounts.bash

```diff
@@ -0,0 +1,273 @@
+#!/bin/bash
+
+#
+# mount helpers
+#
+# Note: these should not use bats, so as to allow their use
+# when manually reproducing tests.
+#
+
+. $(dirname ${BASH_SOURCE[0]})/run.bash
+
+function empty_list() {
```

Comment:
Suggestion: rename `empty_list()` -> `is_list_empty()` ? 

(otherwise it sounds like a statement that says "make this list empty").